en/services/mission.md

```diff
@@ -359,22 +359,20 @@ Mission upload, download, clearing missions, and monitoring progress are support
 > **Note** ArduPilot implements also partial mission upload using `MISSION_WRITE_PARTIAL_LIST`, but not partial mission download (`MISSION_REQUEST_PARTIAL_LIST`).
   Partial mission upload/download is not an official/standardised part of the mission service.
 
-ArduPilot's implementation differs from this specification (non-exhaustively):
+ArduPilot's implementation differs from this specification (non-exhaustively): 
 - The first mission sequence number (`seq==0`) is populated with the home position of the vehicle instead of the first mission item.
 - Mission uploads are not "atomic". 
   An upload that fails (or is canceled) part-way through will not match the pre-update state.
   Instead it may be a mix of the original and new mission.
 - Even if upload is successful, the vehicle mission may not match the version on the uploading system (and if the mission is then downloaded it will differ from the original).
-  - If you try and upload more items than ArduPilot can store the system will "accept" the items (i.e. not report a failure) but will just overwrite each new item to the same (highest) slot in the mission list.
```

Comment:
Note, @peterbarker, you said this is not true so deleted. Can you check the rest of this and confirm if it is accurate? Feel free to push a pr on the docs for this section.